src/pal/src/cruntime/misc.cpp

```diff
@@ -293,23 +293,22 @@ void PAL__mm_setcsr(unsigned int i)
 
 #endif // _AMD64_ 
 
-#if defined(_DEBUG)
```

Comment:
Because there is no way to have conditional exports and PAL_memcpy needs to be exported. I didn’t want to have an almost duplicate second file for debug.